syntax = "proto3";

package MyHistoryDay;

service GCalendarEvents {
    rpc GetEvents(EventsRequest) returns (stream Event) {}
}

message EventsRequest {
    string token = 1;
    string refresh_token = 2;
    string token_uri = 3;
    string client_id = 4;
    string client_secret = 5;
    string scopes = 6;
    string api_service_name = 7;
    string api_version = 8;
    string requested_date = 9;
}

message Event {
    string event_start_time = 1;
    string event_title = 2;
}